test/e2e/metrics_test.go

```diff
@@ -0,0 +1,123 @@
+package e2e
```

Comment:
should this go to library-go as an e2e helper and just a oneliner here?

Reply:
I intend to move to library-go once finalized.

Reply:
I've opened https://github.com/openshift/library-go/pull/596 to start.